walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/HelloWorld.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
```

Comment:
This includes changes from https://github.com/GerardoGran/SPS-GerardoGran/pull/2. Please rebase on merging https://github.com/GerardoGran/SPS-GerardoGran/pull/2 to master